.pre-commit-config.yaml

```diff
@@ -16,3 +16,9 @@ repos:
   rev: 5.12.0
   hooks:
   - id: isort
+- repo: https://github.com/adamchainz/blacken-docs
+  rev: 1.13.0
+  hooks:
+  - id: blacken-docs
+    additional_dependencies:
+    - black==22.12.0
```

Comment:
We will need to switch to `23.1.0` now that https://github.com/scrapy/scrapy/pull/5823 has been merged.